osi_common.proto

```diff
@@ -338,6 +338,45 @@ message Spherical3d
     optional double elevation = 3;
 }
 
+//
+// \brief Assignment of an object to a logical lane
+//
+// An object is assigned to a logical lane if it overlaps the logical lane.
+// Assignment happens even if the reference point is outside the lane, and only
```

Comment:
Is there a definition how large the overlapping area must be for a lane assignment?
Is it possible to read  the size of the overlapping area?

Reply:
Added a note that assignment has to be added as soon as object overlaps more than 5cm.
Size of the overlapping area has to be calculated by the application, but st coordinates help there.